tests/api_connexion/endpoints/test_task_endpoint.py

```diff
@@ -332,6 +336,7 @@ def test_get_tasks_mapped(self):
                     "execution_timeout": None,
                     "extra_links": [],
                     "is_mapped": True,
+                    # todo: should be "outlet_datasets": [], but does not work for mapped
```

Comment:
What do you mean by this?

Reply:
fundamentally, it doesn't make sense to not support datasets just because a task is mapped.  that sorta means mapped tasks aren't first class citizens.  with mapped tasks, we'll soon be able to create "map kwargs" essentially (so iterating over list(dict) instead of list(primitive), which i think should unlock mapping the outlets param too?  i dunno. it needs thought.  but, just doesn't seem right for it to not be supported, for a datasets API -- are mapped tasks not supposed to also be used for data?